gatsby/createPages.js

```diff
@@ -37,12 +37,22 @@ module.exports.createMarkdownPages = async ({ graphql, actions }) => {
 							slug: node.fields.slug,
 							section: node.fields.section,
 						},
-					}
```

Comment:
@SavvyShah We need this routing logic to be intact. Unless this is handled elsewhere, it should be present as earlier.